examples/linear_model/plot_sgd_early_stopping.py

```diff
@@ -47,7 +47,8 @@
 import pandas as pd
 import numpy as np
 import matplotlib.pyplot as plt
-
+import matplotlib
+from sklearn.externals._packaging.version import parse as parse_version
```

Comment:
even though it's the same, we tend to import it from fixes in other files. Let's be consistent
```suggestion
from sklearn.utils.fixes import parse_version
```